packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -134,6 +134,243 @@ describe('scriptUtils', () => {
         })
     })
 
+    describe('getLowestPackageVersion', () => {
+        test('should work when major version is different', async () => {
+            const dependencyTree = {
```

Comment:
These dependency trees take up a lot of lines. Could you move them to a separate file for mocks? (Alternatively, make a `createDependencyTree` helper, but I don't think they can be cleaned up in a way that doing that would be worthwhile.)